src/components/NFTPage/CollectorList/index.vue

```diff
@@ -157,6 +124,15 @@ export default {
       default: false,
     },
   },
+  data() {
+    return {
+      labelTexts: [
```

Comment:
can move to computed as the text does not change, and rename to  `tableHeaderItems`?